command/loadbot/loadbot_command.go

```diff
@@ -433,6 +440,11 @@ func (lr *LoadbotResult) Output() string {
 		fmt.Sprintf("Transactions failed|%d", lr.CountData.Failed),
 	}))
 
+	buffer.WriteString("\n\n[REAL TPS]\n")
```

Comment:
Please rename this section to something like `APPROXIMATE TPS`

Reply:
Renamed f9886aed1bef4a43315d3b30f970cda6e1c9e8c2